lib/parse/slackMessage.js

```diff
@@ -4,29 +4,38 @@ const fundType = require('../constant/fundType');
 function getActionFromMessage(message) {
   const upperMessage = message.toUpperCase();
   const includesGraph = upperMessage.includes('GRAPH');
+  const includesInvested = upperMessage.includes('INVESTED');
+  const includesTotal = upperMessage.includes('TOTAL');
   /*
-  const isGreeting = [
-    'HI',
-    'HEY',
-    'HELLO',
-    'YO',
-    'WHAT\'S UP',
-    'WHAT UP',
-    'WHADDUP',
-    'WADDUP',
-    'SUP',
-    'HOW ARE',
-    'DOING',
-    'GOING ON',
-    'HOWDY',
-  ].some(greeting => upperMessage.includes(greeting));
-  */
-
-  return includesGraph ? action.graph : action.latestValuation;
+   const isGreeting = [
+   'HI',
+   'HEY',
+   'HELLO',
+   'YO',
+   'WHAT\'S UP',
+   'WHAT UP',
+   'WHADDUP',
+   'WADDUP',
+   'SUP',
+   'HOW ARE',
+   'DOING',
+   'GOING ON',
+   'HOWDY',
+   ].some(greeting => upperMessage.includes(greeting));
+   */
+  if (includesGraph) {
+    return action.graph;
+  } else if (includesInvested) {
```

Comment:
Wouldn't it make sense to have the `total` check before the `includesInvested` check?

Then if someone writes `@investbot what is the invested total` it would reply with the total instead of a single fund?